homework/generate-sequence/generateSequence.hpp

```diff
@@ -2,6 +2,17 @@
 #include <vector>
 
 std::vector<int> generateSequence(int count, int step) {
-    // TODO: Implement me :)
-    return {};
+    std::vector<int> result;
+    
+    if(count <= 0)
+    {
+        return result;
+    }
```

Comment:
Try to figure it out, whether you need this condition? Check what you have in `for` loop.

Reply:
I know, that you want to avoid reserve memory if you have negate value. You could move this if on he top and return {};

```C++
if (count <= 0) {
    return {};
}

std::vector<int> vec(count); // will resize vector and initialize with default values.
````

<!--StartFragment-->    if(count &lt;= 0)    {        return result;    }<!--EndFragment-->



<!--StartFragment-->    if(count &lt;= 0)    {        return result;    }<!--EndFragment-->